### PR TITLE
Minor cleanup to the Windows page

### PR DESCRIPTION
**Pull request by tas50**
This page needs to be reworked heavily, but this removes a few things that were outdated.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: chef_master/source/windows.rst [hunk 1]
```diff
@@ -11,8 +11,7 @@ The chef-client has specific components that are designed to support unique aspe
...
 * Use the **dsc_resource** to use Powershell DSC resources in Chef!
 * Two knife plugins dedicated to the Microsoft Windows platform are available: ``knife azure`` is used to manage virtual instances in Microsoft Azure; ``knife windows`` is used to interact with and manage physical nodes that are running Microsoft Windows, such as desktops and servers
-* Four cookbooks provide application-specific support. For `PowerShell 4.0 <https://github.com/chef-cookbooks/powershell>`_. For `IIS 7.0/7.5/8.0 <https://github.com/chef-cookbooks/iis>`_. For `SQL Server <https://github.com/chef-cookbooks/database>`_. And for configuring various settings and behaviors on a machine that is running `Windows <https://github.com/chef-cookbooks/windows>`_
-* Support for both :i386 and :x86_64 architectures
+* Many community cookbooks on Supermarket provide Windows specific support. Chef maintains `PowerShell <https://github.com/chef-cookbooks/powershell>`_, `IIS <https://github.com/chef-cookbooks/iis>`_, `SQL Server <https://github.com/chef-cookbooks/database>`_, and `Windows <https://github.com/chef-cookbooks/windows>`_ cookbooks.
```
Comment (tas50) [inline chef_master/source/windows.rst]: I understand we can do better here. This whole section needs a rewrite. This is a step in the right direction though. Previously it sounded like we only had 4 Windows cookbooks.

Comment (thommay) [reply chef_master/source/windows.rst]: needs a "the" after maintains, but definitely better.

**Review by thommay — CHANGES_REQUESTED** [review]

**Comment by tas50** [comment]
I was thinking of the list as a non-definitive list, but I think that isn't coming across so I'll add 'the' there. In my mind our windows cookbook isn't 'the' windows cookbook. Someone else is more than welcome to make another one.

**Comment by davidwrede** [comment]
@tas50 You're done iterating on this PR, correct? If so, I'll get it merged in shortly.

**Comment by tas50** [comment]
I'm done with this doc for now

**Comment by davidwrede** [comment]
gif-keyboard-4144949959141796710

**Pull request merged**